huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_partition_v3_test.go

```diff
@@ -0,0 +1,184 @@
+package cce
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk/openstack/cce/v3/partitions"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+)
+
+func TestAccCCEPartitionV3_basic(t *testing.T) {
+	var partition partitions.Partitions
+
+	rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+	updateName := rName + "update"
+	resourceName := "huaweicloud_cce_partition.test"
+	// clusterName here is used to provide the cluster id to fetch cce partition.
+	clusterName := "huaweicloud_cce_cluster.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
```

Comment:
add a PreCheck for the partition, as the partition resource can't be used in most situations

Reply:
OK, add `acceptance.TestAccPreCheckCcePartitionAz` to preCheck for the partition